bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -109,13 +155,15 @@ private void initSqlTypes() {
         sqlTypes.put("CALLITEM", "VARCHAR(200)");
         sqlTypes.put("COLORITEM", "VARCHAR(70)");
         sqlTypes.put("CONTACTITEM", "VARCHAR(6)");
-        sqlTypes.put("DATETIMEITEM", "DATETIME");
+        sqlTypes.put("DATETIMEITEM", "TIMESTAMP");
```

Comment:
I am concerned that existing JDBC persistence users, who have existing databases, will upgrade to a release that contains this PR, then the user will add items to their persistence, and their database will contain tables created with DATETIME columns before this PR, and other tables created with TIMESTAMP columns created after their upgrade.  Please let me know what the compatibility issues are for this PR.